src/types/utils.cpp

```diff
@@ -284,6 +285,791 @@ static constexpr std::array<til::color, 256> standardXterm256ColorTable{
     til::color{ 0xEE, 0xEE, 0xEE },
 };
 
+static til::static_map<std::wstring, til::color, std::less<std::wstring>, 782> xorgAppColorTable{

```

Comment:
We may be able to simplify this and remove the whole template pack -- we won't need to include the number of entries or anything.

Check out the string_view map here:

https://github.com/microsoft/terminal/blob/88d1527985123aea66537447241e3fa989fbf755/src/til/ut_til/StaticMapTests.cpp#L58-L63

Reply:
This would be an improvement, but I can't help thinking that a regular `std::unordered_map` would be more efficient than this. It's not like we're getting compile-time construction of the map here, so what is the advantage of the `static_map` in this case?